OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -157,6 +159,11 @@ public void ResolveOrder(Actor self, Order order)
 				self.QueueActivity(new DeployForGrantedCondition(self, this));
 		}
 
+		public string VoicePhraseForOrder(Actor self, Order order)
+		{
+			return (order.OrderString == "GrantConditionOnDeploy") ? Info.Voice : null;
```

Comment:
Style nit: The parens aren't needed here.

Reply:
Fixed the style nit and also fixed the same nit in `Transforms`.

Reply:
Not fixed yet.

Reply:
Apologize again, another misunderstanding.